mailbox/api/src/main/java/org/apache/james/mailbox/model/search/MailboxQuery.java

```diff
@@ -95,7 +96,7 @@ public Builder expression(MailboxNameExpression expression) {
         }
         
         public Builder matchesAllMailboxNames() {
-            this.mailboxNameExpression = Optional.of(new Wildcard());
+            this.mailboxNameExpression = Optional.of(WILDCARD);
```

Comment:
it's not what I meant, Wildcard whould not have a public constructor as it's always the exact same object.

Reply:
Could you handle this?